config/galaxy.ini.sample

```diff
@@ -1031,6 +1031,11 @@ use_interactive = True
 # if running many handlers.
 #cache_user_job_count = False
 
+#Extra directories to be mounted inside Docker containers. Note that ALL docker containers
+#will mount these volumes
+#extra_docker_volumes = /opt/Genomics:ro,/cluster_share/Galaxy/database/results:rw
```

Comment:
@kgururaj - Does this same variable need to be present in the universe_wsgi.ini file as mentioned in https://github.com/Intel-HSS/galaxy/pull/30/files